### PR TITLE
Quote of the day

### PR DESCRIPTION
**Pull request by KevinBoyse**
Please review this, @casey-collab

### CODE REVIEW

File: english/quote-of-the-day/20150314/20150314.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The delights of self-discovery are always available.
```
Comment (casey-collab) [inline english/quote-of-the-day/20150314/20150314.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/quote-of-the-day/20150314/life.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The delights of self-discovery are always available.
```
Comment (casey-collab) [inline english/quote-of-the-day/20150314/life.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline english/quote-of-the-day/20150314/life.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**